test/main/deepEqual.js

```diff
@@ -1581,6 +1581,35 @@ QUnit.test( "Test that must be done at the end because they extend some primitiv
 	}
 );
 
+QUnit.test( "Compare primitive objects", function( assert ) {
```

Comment:
This should also be called "Compare primitive values", they are not primitive objects.